src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -57,15 +171,46 @@ public void exec(final URI home) throws IOException {
                         new RsPrint(
                             new TkProxy(
                                 String.format(
-                                    "%s:%d",
-                                    home.getHost(), home.getPort()
-                                )
+                                    FORMAT, home.getHost(), home.getPort()
```

Comment:
@aschworer move formatting into static method, which receive host and port. Thus, you will get rid of one more static field `FORMAT`